.github/workflows/ci.yml

```diff
@@ -103,7 +108,7 @@ jobs:
         # Only run on non-PR events or only PRs that aren't from forks
         if: github.event_name != 'pull_request' || github.event.pull_request.head.repo.full_name == github.repository
         env:
-          GITHUB_TOKEN: ${{ secrets.GH_TOKEN || secrets.GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           SEMANTIC_RELEASE_PACKAGE: ${{ github.event.repository.name }}
```

Comment:
With branch protection in `main`, the release will fail this way. You'll need to modify the Semantic Release step to use our new release bot. You can do this like so: https://github.com/hirosystems/stacks-blockchain-api/commit/d982c793df6c2ff90a9aa4399f2e52808c35629f

Reply:
How does it looks now @CharlieC3?

Reply:
Looks good! Thanks!